web/client/reducers/searchconfig.js

```diff
@@ -19,7 +19,7 @@ function searchconfig(state = null, action) {
             [action.property]: action.value
         });
     case MAP_CONFIG_LOADED: {
-        const textSearchConfig = action.config.map.text_serch_config;
+        const textSearchConfig = action.config.map.text_search_config;
```

Comment:
probably some resources has been saved as text_serch_config :(. 
Maybe is a good Idea if we make old saved maps working. (you should test an old config, but putting (and adding a test)  here, something like : 
```suggestion
        const textSearchConfig = action.config.map.text_search_config || action.config.map.text_serch_config ;
```
may do the job